.github/workflows/digital_ocean_deployment.yaml

```diff
@@ -8,43 +10,41 @@ on:
   workflow_dispatch:
     manual: true
 
+
 jobs:
   deploy:
     name: Deployment
     runs-on: ubuntu-latest
     steps:
+      - name: Install Go
+        run: |
+          sudo apt update
+          sudo apt install -y golang
+
       - name: Checkout
         uses: actions/checkout@v4
 
       - name: Login to Docker Hub
         uses: docker/login-action@v1
         with:
-          username: ${{ secrets.DOCKER_HUB_USERNAME }}
-          password: ${{ secrets.DOCKER_HUB_PASSWORD }}
+          username: ${{ secrets.VIKTORIA_DOCKER_USERNAME }}
```

Comment:
Can you change these to DOCKER_HUB_USERNAME and PASSWORD please.

Reply:
sure